### PR TITLE
Install root dependencies before demeteorizing

### PR DESCRIPTION
**Pull request by jackboberg**
rebuild dependencies if node version changes during build

### CODE REVIEW

**Comment by musgravejw** [comment]
👍

**Comment by franvarney** [comment]
👍

**Comment by ykptke** [comment]
:+1:

File: bin/build [hunk 1]
```diff
@@ -77,6 +83,8 @@ NPM_VERSION=$(get-version --engine npm $PACKAGE_PATH 2>/dev/null) || true
 
 if [[ 'v$NODE_VERSION' != $(node -v) ]]; then
   nvm install $NODE_VERSION > /dev/null 2>&1
+  nvm use $NODE_VERSION
```
Comment (theworkflow) [inline bin/build:86]: Maybe a message saying `Rebuilding node modules` since it may take some time depending on how many dependencies a project has.

**Comment by theworkflow** [comment]
Besides comment, 👍

**Pull request merged**